test/journeys/specs/space/startup-settings.js

```diff
@@ -37,6 +37,27 @@ describe('Space Widget Startup Settings Tests', () => {
     });
   });
 
+  describe('startCall', () => {
+    it('call is placed after startCall is set to true', () => {
```

Comment:
Keep the test, but skip it, because we can't verify if it works or not due to issues with services at the time of my review
```suggestion
    it.skip('call is placed after startCall is set to true', () => {
```